Marlin/src/pins/stm32f4/pins_RUMBA32_common.h

```diff
@@ -165,6 +165,11 @@
     #define LCD_PINS_D5                     PE13
     #define LCD_PINS_D6                     PE14
     #define LCD_PINS_D7                     PE15
+
+    #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
+      #define BTN_ENC_EN             LCD_PINS_D7  // Detect the presence of the encoder
```

Comment:
How confident are we that all these platforms have pullup-capable pins, to avoid locking out encoder input from real REPRAP_DISCOUNT displays when they are connected?

I'm not sure if that is a universal capability on every pin or not.

Reply:
the input pullups are already in use for other functions like encoder dir pins and button.